rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildExecutionConfigurationRest.java

```diff
@@ -95,6 +95,18 @@ public BuildExecutionConfiguration toBuildExecutionConfiguration() {
         );
     }
 
+    public BuildConfigurationAuditedRest createBuildConfigurationAuditedRest() {
```

Comment:
Now it aligns with other *REST DTOs. Ok to merge.
We are going to revisit this soon as we plan to decouple rest-model from db-model.